includes/HookHandlers/RequestWiki.php

```diff
@@ -468,10 +459,7 @@ public function onRequestWikiQueueFormDescriptorModify(
 			newField: [
 				'type' => 'multiselect',
 				'label-message' => 'requestwiki-label-defaultextensions',
-				'options' => array_combine(
-					$this->options->get( 'MirahezeMagicRequestWikiExtensions' ),
-					$this->options->get( 'MirahezeMagicRequestWikiExtensions' )
-				),
+				'options' => $this->options->get( 'MirahezeMagicRequestWikiExtensions' ),
```

Comment:
Update the default value for `$wgMirahezeMagicRequestWikiExtensions` to an object instead of an array

Reply:
It is an array though even an object is an array in the config. Using {} doesn't matter and makes it less clear what it is actually doing.

Reply:
Oops, I perhaps should've worded that more clearly. I mean to update the default value in extension.json to an empty object

Reply:
Thats what I mean also. An empty object is invalid in the syntax of the manifest IIRC (can't remember for sure) but in the config in manifest an object = an associative array. It is equivalent to $wgConfig = [ 'key' => 'value' ] thus it is still an array. Using an empty object even if it actually works makes it actually less clear that we are actually using an array there.